### PR TITLE
Remove global ia namespace from components

### PR DESCRIPTION
**Pull request by caseysinclair**
This a delete and rename PR along with some minor adjustments to the Layout demo component.

Work completed: 
- Deleted the legacy layout and navbar components
- Removed `global-ia` namespace from the newer components
- New components inherit the same namespace `Layout` and `NavigationBar`

### CODE REVIEW

**Comment by caseysinclair** [comment]
[Handover navigation component to Design Systems](https://trello.com/c/ofRmx4jn/969-handover-navigation-component-to-design-systems)

File: components/Layout/Layout.module.scss [hunk 1]
```diff
@@ -7,7 +7,9 @@
 .navigationBar {
   @extend %navigation-bar;
   box-sizing: border-box;
-  width: 60px;
+  position: sticky;
```
Comment (rodleviton) [inline components/Layout/Layout.module.scss]: These look like new additions??

File: components/Layout/_styles.scss [hunk 2]
```diff
@@ -38,24 +19,36 @@
   display: flex;
```
Comment (rodleviton) [inline components/Layout/_styles.scss]: just making sure this is not a copy and paste error. These styles were applied to `global-ia-page` previously.

File: components/NavigationBar/NavigationBar.js [hunk 3]
```diff
@@ -10,27 +11,23 @@ import {
...
-  | React.Element<typeof Menu>
-  | React.Element<typeof Tooltip>;
+type SupportedChild = React.Element<typeof Link> | React.Element<typeof Menu>;
 
 type Props = {|
   environment: string,
   loading: boolean,
   colorScheme: 'cultureamp' | 'kaizen',
+  badgeHref: string,
+  footerComponent?: ?React.Node,
```
Comment (rodleviton) [inline components/NavigationBar/NavigationBar.js]: Shouldn't this be `footerComponent?: React.Node`?

**Pull request closed without merging**